redun/executors/aws_utils.py

```diff
@@ -59,6 +61,38 @@ def get_aws_client(service: str, aws_region: str = DEFAULT_AWS_REGION) -> boto3.
     return client
 
 
+def is_ec2_instance() -> bool:
```

Comment:
I brought this over as well from `aws_batch.py`, since its needed in the function below.